src/ProjectSystem/Impl/Project/FileSystemMirroringProjectFileGenerator.cs

```diff
@@ -63,6 +64,7 @@ public abstract class FileSystemMirroringProjectFileGenerator : IVsProjectGenera
                             new XProjElement("UserProperties")
                         )
                     ),
+                    new XProjElement("Target", new XAttribute("Name", "Build")),
```

Comment:
Does this mean F5/CTRL-F5 won't work for existing projects?  If yes, what is the symptom?  Start command disabled?
I'm not saying that it needs to work for existing projects.  We're still in preview after all.  We could just document the manual change.